translation.yml

```diff
@@ -0,0 +1,8 @@
+source_language: en
+target_languages: [cs, da, de, es, fi, fr, hi, it, ja, ko, ms, nb, nl, pl, pt-BR, pt-PT, sv, th, tr, vi, zh-CN, zh-TW]
```

Comment:
```suggestion
target_languages: [bg-BG, cs, da, de, el, es, fi, fr, hi, hr-HR, hu, id, it, ja, ko, lt-LT, ms, nb, nl, pl, pt-BR, pt-PT, ro-RO, ru, sk-SK, sl-SI, sv, th, tr, vi, zh-CN, zh-TW]
```

Reply:
This will fail CI. The locales you've added in no longer seem to be supported.

Reply:
I think we just need to make and adjustment on tp side of things to handle these additional buyer locales

Reply:
https://github.com/Shopify/translation-platform/pull/6186/

Reply:
Updated to the complete list, can re-run the check once the above PR is merged, TY! 😎